src/transformers/modeling_utils.py

```diff
@@ -384,7 +386,7 @@ def floating_point_ops(
         return 6 * self.estimate_tokens(input_dict) * self.num_parameters(exclude_embeddings=exclude_embeddings)
 
 
-class PreTrainedModel(nn.Module, ModuleUtilsMixin, GenerationMixin):
+class PreTrainedModel(nn.Module, ModuleUtilsMixin, GenerationMixin, ModelHubMixin):
```

Comment:
I'm wondering if the `ModelHubMixin` couldn't be in `ModuleUtilsMixin` rather than `PreTrainedModel`?
Also we'll have to think of adding this to TensorFlow, as this is an aspect where parity is important.

Reply:
Ah yes, TensorFlow and Flax should both inherit from the mixin as well.

Reply:
There's a `save_pretrained` in `ModelHubMixin` but I guess it's overriden by the one in `PreTrainedModel`

We might want to implement the `push_to_hub=True` param in `PreTrainedModel`'s `save_pretrained` too, what do you think?